ConfigSpace/hyperparameters.pyx

```diff
@@ -258,6 +258,38 @@ cdef class Constant(Hyperparameter):
                       transform: bool = False) -> List:
         return []
 
+    def pdf(self, vector: np.ndarray) -> np.ndarray:
+        """
+        Computes the probability density function of the parameter in 
+        the original parameter space (the one specified by the user).
+        For each parameter type, there is also a method _pdf which 
+        operates on the transformed (and possibly normalized) parameter
+        space. Only legal values return a positive probability density,
+        otherwise zero.
+        vector: np.ndarray
```

Comment:
Could you please adopt numpydoc docstring formatting?

Reply:
Sure, will fix!